.drone.yml

```diff
@@ -144,6 +144,27 @@ trigger:
   event:
   - push
 
+
+---
+kind: pipeline
+type: exec
+name: integration-tests-jail-unjail
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: integration-tests
+  commands:
+  - export NIX_REMOTE=daemon
+  - export PATH="$PATH:/nix/var/nix/profiles/default/bin/"
+  - export DOCKER_COMPOSE_PREFIX="${DRONE_BRANCH}"
+  - . /etc/profile.d/nix.sh  
```

Comment:
this file may not be on the new CI server

Reply:
ok